appveyor.yml

```diff
@@ -0,0 +1,34 @@
+branches:
+  only:
+    - master
+
+environment:
+  ZEUS_HOOK_BASE:
+    secure: xxx
+  matrix:
+    - nodejs_version: "10"
+    - nodejs_version: "8"
```

Comment:
Let's only test the latest version to keep our number of AppVeyor builds low. We only have two concurrent builds at the moment, and I don't want zeus-cli to have a large impact on that.